src/pages/Manager.vue

```diff
@@ -637,18 +637,20 @@ import ModalCard from '../components/ModalCard.vue';
                     });
                     return;
                 }
-                DownloadModModal.downloadSpecific(this.profile.asImmutableProfile(), combo, this.$store)
-                    .then(async value => {
-                        const modList = await ProfileModList.getModList(this.profile.asImmutableProfile());
-                        if (!(modList instanceof R2Error)) {
-                            await this.$store.dispatch('profile/updateModList', modList);
-                        } else {
-                            this.$store.commit('error/handleError', modList);
-                        }
-                    })
-                    .catch(
-                        (err: R2Error) => this.$store.commit('error/handleError', err)
-                    );
+
+				try {
```

Comment:
Would this be an improvement?

```
				try {
                	await DownloadModModal.downloadSpecific(this.profile.asImmutableProfile(), combo, this.$store);
				    const modList = await ProfileModList.getModList(this.profile.asImmutableProfile());
                    if (modList instanceof R2Error) {
                        throw modList;
                    }
                    await this.$store.dispatch('profile/updateModList', modList);
				} catch (err) {
					this.$store.commit('error/handleError', R2Error.fromThrownValue(err));
				}
```

`throwForR2Error` seems to be imported but not used.

Reply:
I'll fix this in an upcoming cleanup PR 👍